codesign/eclipse-codesign.sh

```diff
@@ -42,14 +42,14 @@ do
 
     case $OPERATING_SYSTEM in
         windows) 
-            curl -o "$f" -F file="@${dir}/unsigned_${file}" https://cbi.eclipse.org/authenticode/sign
+            curl -f -s -S -o "$f" -F file="@${dir}/unsigned_${file}" https://cbi.eclipse.org/authenticode/sign
             ;;
         mac)
-            curl -o "$f" -F file="@${dir}/unsigned_${file}" https://cbi.eclipse.org/macos/codesign/sign
+            curl -f -s -S -o "$f" -F file="@${dir}/unsigned_${file}" https://cbi.eclipse.org/macos/codesign/sign
```

Comment:
the error code are documented in curl and because we use "set +e" the script exit with anything != 0 so the error code can be seen in the caller script or jenkins job.
As of the error string it will be displayed in stderr because of --show-error (-S)
exemple: stderr:
>curl: (22) The requested URL returned error: 404 Not Found

--silent (-s) only hide progress bar/stats etc I think

>  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                 Dload  Upload   Total   Spent    Left  Speed
>100   334    0     0  100   334      0   1815 --:--:-- --:--:-- --:--:--  1815